site/docs/foundations/responsiveness/index.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: Layout grid
+layout: DetailTechnical
+sidebar:
+  priority: 6
+---
+
+A responsive layout grid is made up of a set of consistently proportioned columns, separated by gutters, and flanked by margins. Basic layout grids are included in the design system to provide a starting point for designing any user interface.
+
+The number of columns and their widths vary depending on the density and breakpoint set. Learn more information about Salt densities in the [density foundation](/salt/foundations/density).
+
+<Image
+  src="/img/foundations/responsiveness-gutters.png"
+  alt="Illustration of columns, gutters and margins."
+/>
+
+| Spacing type   | Definition                                                                                                                       | 
+| -------------- | -------------------------------------------------------------------------------------------------------------------------------- | 
+| Columns        | The space that divides the content of the page into numbers depending on the content width for that breakpoint                   |
```

Comment:
Do we still maintain a glossary, these definitions could be there and you could turn above references into links and then remove this.